ods_ci/tests/Resources/Common.robot

```diff
@@ -308,21 +308,21 @@ Run Keyword If RHODS Is Self-Managed
     ${is_self_managed}=    Is RHODS Self-Managed
     IF    ${is_self_managed} == True    Run Keyword    ${name}    @{arguments}
 
-Get Domain From Current URL
-    [Documentation]    Gets the lowest level domain from the current URL (i.e. everything before the first dot in the URL)
+Get Sub Domain Of Current URL
```

Comment:
well, technically is not taking the subdomain only, but the domain + any subdomains